src/ctapipe/image/muon/intensity_fitter.py

```diff
@@ -474,11 +474,14 @@ class MuonIntensityFitter(TelescopeComponent):
     ).tag(config=True)
 
     hole_radius_m = FloatTelescopeParameter(
-        help="Hole radius of the reflector in m",
+        help="The radius of the hole in the center of the primary mirror dish in meters."
+        "The hole has hexagonal shape (same as a single mirror facet)."
```

Comment:
This does not apply to all telescopes (e.g. MAGIC has square facets and the whole is two facets large).

Reply:
I would just describe the configuration value here ( radius of the circle approximation of the central mirror hole)